.vib/spring-cloud-dataflow/vib-verify.json

```diff
@@ -1,36 +1,36 @@
 {
-  "phases": {
```

Comment:
AFAIK, we are leaving this blank space in our `.json` files. Could you please recheck your linter config and re-run it here?

Reply:
I passed this through linter and this is the result. Which linter should I use for `JSON` files and what should be the config?